app/src/main/java/mozilla/lockbox/support/SecurePreferences.kt

```diff
@@ -39,7 +40,8 @@ open class SecurePreferences(
             try {
                 val plain = keystore.decryptBytes(encrypted)
                 String(plain, StandardCharsets.UTF_8)
-            } catch (error: IllegalArgumentException) {
+            } catch (error: Throwable) {
+                log.error("DecryptBytes exception: ", error)
```

Comment:
There are quite a few exceptions possible from this method: would it make sense to be a bit more specific, using multiple catch blocks?

e.g. Some of these may be recoverable, some may be our fault, some may not.